edgecast/resources/rulesengine/v4policy.go

```diff
@@ -449,5 +448,5 @@ func policyDiffSuppress(k, old, new string, _ *schema.ResourceData) bool {
 	delete(oldPolicy, "name")
 	delete(newPolicy, "name")
 
-	return reflect.DeepEqual(oldPolicy, newPolicy) == false
+	return reflect.DeepEqual(oldPolicy, newPolicy) != false
```

Comment:
Ah I should have caught this the first time I reviewed this code. Perhaps just leave it as `return reflect.DeepEqual(oldPolicy, newPolicy)` however. The `!=` is a little bit confusing.